recipes/anaconda-verify/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "anaconda-verify" %}
+{% set version = "1.2.0" %}
+{% set checksum = "e790262f88b3c3ef4ac9d647e0a385ea8ec665ba34509879d0a2270456f5220e" %}
+
+package:
+  name: {{ name.lower() }}
```

Comment:
We don't. I was trying to come up with a PyPI template and this is a holdover from it.